_posts/2023-12-13-gift-guide.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: blog_post
+title: "Compiler's 2023 Holiday Gift Guide"
```

Comment:
```suggestion
title: "Compiler’s 2023 Holiday Gift Guide"
```

Reply:
https://smartquotesforsmartpeople.com/

Reply:
I had no idea this was a thing!

Reply:
To make these suggestion changes, you can click **Commit suggestion** and then **Commit changes**

<img width="798" alt="image" src="https://github.com/compilerla/compiler.la/assets/3673236/f543af4f-59a6-4f0e-98ed-e4497d7eac82">